.jshintrc

```diff
@@ -7,7 +7,6 @@
   "eqeqeq"       : true,
   "eqnull"       : true,
   "immed"        : true,
-  "latedef"      : true,
```

Comment:
I can add this one back, but it will involve making _another_ file, and I don't really want to do that.